src/select.rs

```diff
@@ -6,13 +6,19 @@ use std::ascii::AsciiExt;
 use string_cache::{Atom, Namespace};
 use tree::{NodeRef, NodeData, ElementData};
 
+/// The definition of whitespace per CSS Selectors Level 3 § 4.
+///
+/// Copied from rust-selectors.
+static SELECTOR_WHITESPACE: &'static [char] = &[' ', '\t', '\n', '\r', '\x0C'];
+
+#[derive(Debug)]
 pub struct KuchikiSelectors;
 
 impl SelectorImpl for KuchikiSelectors {
     type NonTSPseudoClass = PseudoClass;
     type PseudoElement = PseudoElement;
```

Comment:
Updated and rearranged typedefs such that they match the `rust-selector` sourcecode.